scripts/create_pr.sh

```diff
@@ -13,30 +13,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-./vuln-list-k8s -vuln-list-dir "$VULN_LIST_DIR"
+./cve-feed-osv
 
-cd "$VULN_LIST_DIR" || exit 1
+cd "cve-feed-osv" || exit 1
 
 if [[ -n $(git status --porcelain) ]]; then
   # List changed files
   CHANGED_FILES=$(git ls-files . --exclude-standard --others | grep "CVE")
-  REPO="$REPOSITORY_OWNER/$VULN_LIST_DIR"
+  REPO="kubernetes-sigs/cve-feed-osv"
```

Comment:
I think we should use "GITHUB_REPOSITORY" env so we can easily test it in our fork (e.g. knqyf263/cve-feed-osv). If it's not set, we use kubernetes-sigs/cve-feed-osv as the default value.
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables